examples/basic/story-wrapper.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react'
 import Frame from 'react-frame-component'
 
-export default ({ children, style = {}, ...rest }) => {
+import StyledFrame from 'react-styled-frame'
+import StyledShadow from 'react-styled-shadow-dom'
+
+export default (props) => {
+  const { wrapper, ...rest } = props
+
+  if (wrapper === 'styledFrame') {
+    return <StyledFrame {...rest} />
+  }
+
+  if (wrapper === 'shadowDOM') {
+    return <StyledShadow {...rest} />
+  }
+
+  const { children, style, theRest } = rest
```

Comment:
theRest will not do what you want here, you want `...theRest`

Reply:
✅